src/transforms/stdnames.jl

```diff
@@ -0,0 +1,73 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(:spec)
+
+The transform that standardizes the column names.
+There are three different methods of standardization: `:upper`, `:camel`, and `:snake`.
+Transform defaults to `:upper`.
+
+# Examples
+
+| Transform | Original Col. Name | Transformed Col. Name |
+| ----------- | ----------- |	----------- |
+| `StdNames`    		 | apple_Trees | APPLETREES |
+| `StdNames(:upper)` | apple_Trees | APPLETREES |
+| `StdNames(:camel)` | apple_Trees | AppleTrees |
+| `StdNames(:snake)` | apple_Trees | apple_trees |
+
+"""
+struct StdNames <: Stateless
+  spec::Symbol
+end
+
+StdNames() = StdNames(:upper)
+
+isrevertible(::Type{StdNames}) = true
+
+function apply(transform::StdNames, table)  
+  cols = Tables.columns(table)
+  oldnames = string.(Tables.columnnames(cols))
+  
+  spec = transform.spec
+
+  spec == :camel && (names = _camel.(oldnames))
+  spec == :snake && (names = _snake.(oldnames))
+  spec == :upper && (names = _upper.(oldnames))
+
+  newnames = _unique(names)
+  oldnew = Dict(zip(oldnames, newnames))
+
+  rtrans = Rename(oldnew)
+  newtable, rcache = apply(rtrans, table)
+
+  newtable, (rtrans, rcache)
+end
+
+function revert(::StdNames, newtable, cache)
+  rtrans, rcache = cache
+  revert(rtrans, newtable, rcache)
+end
+
+const delim = ['_', ' ']
```

Comment:
Can you please move this variable to near where it is used? Right before the _camel function. Please try to think like the reader of the code always.

Reply:
Also, we should probably add `-` to the list of delimiters. So that `day-of-month` gets converted into `DayOfMonth`